target/linux/ath79/dts/ar9342_ubnt_nanobeam-ac.dts

```diff
@@ -51,11 +50,11 @@
 	status = "okay";
 
 	/* default for ar934x, except for 1000M and 10M */
-	pll-data = <0x06000000 0x00000101 0x00001313>;
+	pll-data = <0x02000000 0x00000101 0x00001313>;
 
 	mtd-mac-address = <&art 0x0>;
 
-	phy-mode = "rgmii";
+	phy-mode = "rgmii-id";
```

Comment:
the difference here is probably more worthy of commenting...

Reply:
"In 4.14 the delays were not cleared, so setting "rgmii" as phy-mode did not affect delays set by the bootloader. With 5.4 kernel the situation changed and the ethernet interface stopped working." ;)
That is in the commit message.

Reply:
.... yes.... I can read that too.  How about...

```
In 5.4, phy timings were automatically added by the mac, switch to using the new rgmii-id which uses the old behaviour
```

Could this be worked around by specifying the actual delays instead of just turning off the autosetting?  I don't personally find, "oh, in 5.4 it stopped working, so we changed some magic" to be very descriptive.

Reply:
Okay, I will write a new commit message. ;)
How can I get the rx and tx delays? Just by try and error? Any data sheet, somwhere?

Reply:
```


In 4.14 the delays were not cleared, so setting "rgmii" as phy-mode
did not affect delays set by the bootloader. With 5.4 kernel the
situation changed and the ethernet interface stopped working.

"rgmii" requires rx and tx delays depending on the hardware circuit
and wiring. The mac or the phy can add these delays.
- "rgmii":  delays are controlled by the mac
- "rgmii-id": delays are controlled by the phy
More Information in Linux Kernel Tree:
Documentation/devicetree/bindings/net/ethernet-controller.yaml

"rgmii" should be the preferred mode, which allows the mac layer to
turn off the dealys completely if they are not needed. However, the
delays are not set correctly, which causes the ethernet interface
to be broken. Just taking the ethernetpart from the litebeam ac gen2
will fix the issue.
```

I will try to findout working delays and get rgmii running.

Reply:
@karlp is the commit message okay?